src/sources/multisrc/madara/MadaraGenerator.js

```diff
@@ -257,3 +257,11 @@ export const Novel4UpScraper = new MadaraScraper(
   false,
   2,
 );
+export const TeamXNovelScraper = new MadaraScraper(
+  104,
```

Comment:
`104` is already assigned to a different source. Check for the id in `src/sources/sources.json` and add the source in that file too.

Reply:
Also add `TeamXNovelScraper ` in `src/sources/sources.js`